src/main/python/ui/postbuilder.py

```diff
@@ -203,7 +206,7 @@ def setupUi(self, postbuilder):
         self.ddCheckBox.toggled['bool'].connect(postbuilder.build_avs_post)
         self.dts51CheckBox.toggled['bool'].connect(postbuilder.build_avs_post)
         self.comboBox.currentIndexChanged['int'].connect(postbuilder.post_type_changed)
-        # QtCore.QMetaObject.connectSlotsByName(postbuilder)
+        QtCore.QMetaObject.connectSlotsByName(postbuilder)
```

Comment:
you have to manually comment this out unfortunately whenever you regenerate from the ui files (#304 for details)

Reply:
NB: I have no idea why this causes problems but it seems to be generally unstable and not required so I just comment them out everywhere

Reply:
weird that that causes an issue, it's fixed now